README.md

```diff
@@ -5,14 +5,14 @@ that puma can provide:
 
 Gauges:
 
-* puma.workers - The number of distinct process running. In single mode this will be 1, in cluster mode the number of worker processes
+* puma.workers - The number of distinct processes running. In single mode this will be 1, in cluster mode the number of worker processes
 * puma.old_workers - The number of worker processes that are about to be shut down as part of a phased restart. Will normally be 0
 * puma.booted_workers - The number of worker processes that are in a booted state
-* puma.running - The number of worker threads currently running. In quiet times, idle threads may be shutdown, but they'll start back up again when traffic arrives
-* puma.backlog - The number of requests that have made it to a worker but are yet to be processed. This will normally be zero, as requests queue on the tcp/unix socket in front of the master puma process, not in the worker thread pool
-* puma.pool_capacity - The number of idle and unused worker threads. When this is low/zero, puma is running at full capacity and might need scaling up
+* puma.running - The number of worker threads currently running. In quiet times, idle threads may be shutdown if min threads is less than max threads, but they'll start back up again when traffic arrives
```

Comment:
https://github.com/puma/puma/blob/13e18e8078c800adfc52af687acc1d8de5f3988d/lib/puma/thread_pool.rb#L264

Reply:
Hmm. This extra wording is confusing me a little.

Is this saying that if the current config has set min threads to be lower than max threads, then idle threads will eventually terminate until the running threads equals min threads?